docs/developer-docs/latest/development/admin-customization.md

```diff
@@ -227,7 +227,34 @@ Note also that the main navigation logo uploaded via the admin panel supersedes
 
 #### Favicon
 
-To update the favicon, put a favicon file in the `./src/admin/extensions` folder and update the `config.head.favicon` key in the [admin panel configuration](#configuration-options).
+Replacing the Strapi favicon with a custom favicon.
+:::tip
+This same process may be used to replace the login logo `AuthLogo` and menu logo with `MenuLogo`. For more details, please visit Strapi [logos subsection](#logos)
+:::
+
+1. (_optional_) Create a `./src/admin/extensions/` folder if the folder does not already exist.
+2. Upload your favicon into `src/admin/extensions/`.
+3. Replace the **favicon.ico** at`Strapi app root` with custom favicon.ico
+4. Update your `src/admin/app.js` with the following:
+```js
+// path: src/admin/app.js
+
+import favicon from './extensions/favicon.png';
+
+export default {
+  config: {
+         // replace favicon with a custom icon
+         head: {
+                favicon: favicon,
+        },
+  }
+}
+```
+5. Rebuild, launch and revisit your Strapi app `yarn build && yarn develop`.
+
+:::note
+Be certain that the cached favicon is cleared. It can be cached in your web browser and also with your domain management tool like Cloudflare's CDN
```

Comment:
```suggestion
Be certain that the cached favicon is cleared. It can be cached in your web browser and also with your domain management tool like Cloudflare's CDN.
```